api/spec/models/applicant_spec.rb

```diff
@@ -2,45 +2,52 @@
 
 describe Applicant do
   it 'should have a valid factory' do
-    FactoryBot.create(:applicant)
+    a = FactoryBot.create(:applicant)
   end
 
   it 'should not be valid without a first name' do
+    skip "To do. Old tests are commented so fix for new"
```

Comment:
Are these cases work for another issue/PR?

Reply:
Yeah I'll add an issue to do.